okhttp/src/commonMain/kotlin/okhttp3/internal/-HeadersCommon.kt

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright (C) 2022 Square, Inc.
```

Comment:
What are the copyright considerations given above is 

> Licensed to the Apache Software Foundation

Reply:
ASF is a legal entity that owns all the code originally forked from Apache Harmony.
Easily confused with the Apache license, which is a grant from a legal entity.

(If the ASF owns copyright on things, let’s not accidentally lose track of that fact.)